src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/TransferTonnage.cshtml

```diff
@@ -178,7 +179,10 @@
         </table>
 
         <div class="govuk-!-margin-top-3">
-            @this.WeeeGds().Button("Continue", new { id = "transfer-tonnage-button", @class = "govuk-!-width-one-quarter" })
+            <div class="govuk-button-group">
+                @*@this.WeeeGds().Button("Submit evidence note", new { name = "Action", value = ActionEnum.Submit, id = "submit-note", @class = "govuk-!-width-one-quarter" })*@
```

Comment:
that seems commented out? still needed?

Reply:
Will be put back in for the submit story